account_vat_period_end_statement/views/report_vatperiodendstatement.xml

```diff
@@ -137,7 +137,7 @@
                     <table class="table table-condensed">
                             <t t-foreach="statement.payment_ids" t-as="payment_line">
                                 <tr>
-                                    <td t-esc="'Importo versato (Estremi del versamento: data '+ time.strftime('%d/%m/%Y', time.strptime(payment_line.date, '%Y-%m-%d')) + ' - ' +payment_line.journal_id.name+')'" />
+                                    <td t-esc="'Importo versato (Estremi del versamento: data '+ time.strftime('%d/%m/%Y', time.strptime(str(payment_line.date), '%Y-%m-%d')) + ' - ' +payment_line.journal_id.name+')'" />
```

Comment:
Essendo `payment_line.date` una data, è inutile convertirla a stringa per poi convertirla a data per poi convertirla di nuovo a stringa.
Dovrebbe essere sufficiente usare `time.strftime('%d/%m/%Y', payment_line.date)`

Reply:
@eLBati,
non mi torna, provando in una console di Python ottengo:

```
>>> import time, datetime
>>> _  = datetime.date.today()
>>> _
datetime.date(2020, 2, 17)
>>> time.strftime('%d/%m/%Y', _)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument must be 9-item sequence, not datetime.date
>>> time.strftime('%d/%m/%Y', _.timetuple())
'17/02/2020'
```

Reply:
Giusto, bisogna usare `payment_line.date.strftime('%d/%m/%Y')`.
@jado95 puoi verificare?